var/spack/repos/builtin/packages/eztrace/package.py

```diff
@@ -6,21 +6,77 @@
 from spack import *
 
 
-class Eztrace(AutotoolsPackage):
+class Eztrace(Package):
     """EZTrace is a tool to automatically generate execution traces
        of HPC applications."""
 
-    homepage = "https://eztrace.gforge.inria.fr"
-    url      = "https://gitlab.com/eztrace/eztrace/-/archive/eztrace-1.1-10/eztrace-eztrace-1.1-10.tar.gz"
+    homepage = "https://gitlab.com/eztrace"
     maintainers = ['trahay']
+    git = "https://gitlab.com/eztrace/eztrace.git"
 
-    version('1.1-10', sha256='97aba8f3b3b71e8e2f7ef47e00c262234e27b9cb4a870c85c525317a83a3f0d4')
+    version('master',  branch='master')
+    version('2.0-rc1', sha256='2ad322df5aeb0c668ebadbfeb3b21c8841831b2c15944d86d1d217f738fee74e')
+    version('1.1-13', sha256='6144d04fb62b3ccad41af0268cd921161f168d0cca3f6c210c448bb0b07be7e0')
+    version('1.1-10', sha256='63d1af2db38b04efa817614574f381e7536e12db06a2c75375d1795adda3d1d8')
 
     depends_on('mpi')
+    depends_on('opari2')
+    depends_on('binutils')
 
+# eztrace 1.x dependencies
+    depends_on('autoconf@2.71', type='build', when="@:2.0")
+    depends_on('automake', type='build', when="@:2.0")
+    depends_on('libtool', type='build', when="@:2.0")
     # Does not work on Darwin due to MAP_POPULATE
-    conflicts('platform=darwin')
+    conflicts('platform=darwin', when='@:2.0')
```

Comment:
If it conflicts for all 1 versions and not for any of the 2 versions, write this as:
```suggestion
    conflicts('platform=darwin', when='@:1')
```